test/utils/dependency-initialiserSpec.js

```diff
@@ -27,11 +26,74 @@ describe( 'dependency-initialiser', function(){
 		dependencyBInitialiser.on( 'ready', readySpy );
 
 		options.pluginB.setReady();
-		
+
 		setTimeout( function() {
 			expect( options.logger.lastLogEvent ).toBe( 'INFO' );
 			expect( readySpy.calls.count() ).toBe( 1 );
 			done();
 		}, 5 );
 	});
+});
+
+describe( 'encounters timeouts and errors during dependency initialisations', function(){
+	var dependencyInitialiser;
+	var readySpy;
+	var onReady = jasmine.createSpy( 'onReady' );
+	var exit = jasmine.createSpy( 'exit');
+	var log = jasmine.createSpy( 'log' );
+	var originalProcessExit = process.exit;
+	var originalConsoleLog = console.log;
+	var options = {
+		plugin: new PluginMock( 'A' ),
+		logger: { log: jasmine.createSpy( 'log' ), isReady: true },
+		dependencyInitialisationTimeout: 1
+	};
+
+	it( 'disables process exit', function(){
```

Comment:
They used to be - it breaks (try it if you like). Something is different in their timing, it is called before process.exit is called,